src/components/QuranReader/index.tsx

```diff
@@ -157,6 +158,14 @@ const getRequestKey = (
       translations: selectedTranslations.join(', '),
     });
   }
+  if (isQuranicPageData) {
+    return makePageVersesUrl(id, {
+      page,
+      wordFields: `verse_key, verse_id, page_number, location, ${quranReaderStyles.quranFont}`,
+      translations: selectedTranslations.join(', '),
+    });
+  }
+
```

Comment:
Why is there a specific call for Quranic page data? Is it not request translations in the payload?